### ISSUE TITLE
Lose focus when switching tabs

### ISSUE THREAD

**Issue by deeperx** [atom/tree-view#858]
If ```tree-view``` package disabled, it works fine, but if it's not, when switching between tabs when pressing ```cmd+tab``` it loses the cursor focus. I'd needed to click with my mouse to get cursor focus back.

![lose-focus-when-switching-tabs](https://cloud.githubusercontent.com/assets/621232/15755533/e59b46d6-2905-11e6-9d9d-6340d4ce7f46.gif)

**Comment by Ben3eeE** [comment]
It looks like you have auto reveal set to true, and in combination with focus on reveal this is probably expected. See https://github.com/atom/tree-view/issues/850.

**Comment by deeperx** [comment]
Hey @Ben3eeE Thanks for that. That is the same problem. But, it's annoying although it's expected.

**Comment by Ben3eeE** [comment]
Yeah, I'm not sure what the intended behavior is since changing tabs in any other way does not make the tree-view steal focus but the combination of these two settings sounds like it should produce this behavior even if it is a really annoying one.
I accidentally stumbled upon this myself because Focus on reveal is the default option I think, and only effective when you run `Tree View:Reveal Active File`.

**Comment by lee-dohm** [comment]
I've added some repro steps to make the description more clear. Given that there isn't a better behavior proposal, I'm inclined to close this as `wontfix`.

**Comment by lee-dohm** [comment]
The original PR for this functionality https://github.com/atom/tree-view/pull/715 indicates that "Focus On Reveal" was only intended to affect the behavior of the `tree-view:reveal-active-file` command. So I've classified this as a bug.

**Comment by vraravam** [comment]
My usecase is: I want the tree-view to be open all the time, but with the "auto-reveal" feature, the tree-view captures focus. When I open a file or switch the active text editor, all I was to happen is to reveal the file, but the focus should still remain on the text editor. Also, unless I have the "Focus on reveal" is checked, my expected behavior is that the tree-view did not get into the stack of previous focused elements. This is evident if i do `ctrl + tab` to switch based on editor history.
